lib/node_modules/@stdlib/math/base/ops/cnegf/lib/native.js

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var Complex64 = require( '@stdlib/complex/float32' );
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Negate a single-precision complex floating-point number.
+*
+* @private
+* @param {Complex64} z - complex number
+* @returns {Complex64} result
+*
+* @example
+* var Complex64 = require( '@stdlib/complex/float64' );
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* var z1 = new Complex64( -4.2, 5.5 );
+* // returns { 're': -4.2, 'im': 5.5 }
```

Comment:
```suggestion
* // returns <Complex64>
```

We should be returning complex number instances here as per L89.